developers/skale-chains.adoc

```diff
@@ -0,0 +1,176 @@
+= SKALE Chains {revnumber}
+:doctype: article
+:revnumber: v3.5.8
+:revdate: 2021-03-03
+:icons: font
+:toc: macro
+
+ifdef::env-github[]
+
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+
+toc::[]
+
+endif::[]
+
+A SKALE Chain is a Web3 Ethereum compatible elastic blockchain designed to make Web3 cloud computing easier for dApp developers. SKALE Chains (schains) provide EVM compute power, decentralized database storage, and other functionality to help not only power your dApps but allow them to scale in a cost-efficient manner. 
+
+== What makes a SKALE Chain
+
+Schains are powered by 16 out of more than 150 nodes run by over 45 SKALE Network validators. Each schain is secured by Proof-of-Stake from the validator and delegator communities.
+
+Every node in the SKALE Network dynamically supplies containerized Web3 compute resources that work in concert with other nodes. A schain involves 16 randomly chosen nodes that each run a containerized network client with EVM (skaled). All 16 of these skaled containers broadcast network messages, share transactions and blocks, and other cryptographic data with each other. Think of these containers as powerful instances of an Ethereum network, receiving Web3 transactional messages from clients and end-users, organizing them into blocks using an ultra-fast mathematically proven consensus called Asynchronous Binary Byzantine Agreement.
+
+== How to work with a SKALE Chain
+
+Each SKALE Chain powers a single dApp, providing dedicated Web3 compute resources to support dApp EVM execution and transactions from end-users.
```

Comment:
`Each SKALE Chain powers a single dApp` - will lead to questions about if a dApp developer can deploy multiple applications on one chain. Maybe rephrase this so that we can make dApps know they can run many applications on one chain?